src/fetchtastic/client_app_config.py

```
@@ -0,0 +1,223 @@
+        return _dedupe(_as_list(config.get("SELECTED_APP_ASSETS")))
+
+    apk_assets = expand_apk_selected_patterns(
+        _as_list(config.get("SELECTED_APK_ASSETS"))
+    )
+    desktop_assets = _as_list(
+        config.get("SELECTED_DESKTOP_ASSETS", config.get("SELECTED_DESKTOP_PLATFORMS"))
+    )
+    return _dedupe([*apk_assets, *desktop_assets])
+
+
+def normalize_client_app_config(config: dict[str, Any]) -> dict[str, Any]:
```

Ruff (0.15.12)

[warning] Too many statements (53 > 50)

(PLR0915)